hibernate-core/src/main/java/org/hibernate/boot/model/source/internal/hbm/ModelBinder.java

```diff
@@ -523,6 +523,7 @@ private void bindDiscriminatorSubclassEntities(
 			PersistentClass superEntityDescriptor) {
 		for ( IdentifiableTypeSource subType : entitySource.getSubTypes() ) {
 			final SingleTableSubclass subEntityDescriptor = new SingleTableSubclass( superEntityDescriptor, metadataBuildingContext );
+			subEntityDescriptor.setCached(superEntityDescriptor.isCached());
```

Comment:
We have code format requirement (see https://github.com/hibernate/hibernate-ide-codestyles). So it would be appreciated if you add space chars as following to the above statement
```
subEntityDescriptor.setCached( superEntityDescriptor.isCached() );
```
The same applies to the other code changes below.

Reply:
You can require @sebersole to review for he implemented HHH-12146. However, he is super busy  with v6 and patience would be appreciated.

Reply:
Thanks Nathan for your input! 

Fixed code formatting issue. I requested @sebersole as reviewer as suggested, and I totally understand if this is not a top priority on the core teams list! :)